contrib/perf/get-baseline-run.py

```
@@ -0,0 +1,156 @@
+        'Accept': 'application/vnd.github.v3+json'
+    }
+
+    params = {
+        'event': event,
+        'status': 'success',
+        'per_page': limit
+    }
+
+    print(f"Searching for baseline run (event={event}, status=success)...", file=sys.stderr)
+
+    response = requests.get(url, headers=headers, params=params)
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)